doc/specification.md

```diff
@@ -35,62 +35,126 @@ TLS
 
 # 4 Service Definition
 
-The gRIBI service is defined as a single gRPC service, with three RPCs:
+The gRIBI service is a single gRPC service currently defined in [`gribi.proto`](https://github.com/openconfig/gribi/blob/master/v1/proto/service/gribi.proto). It includes three RPCs:
```

Comment:
Is the intention of "currently defined in" because we might move to some v2 at some point in the future? I suspect that we can just say "is defined in".